pkg/volume/gcepd/attacher_test.go

```diff
@@ -284,9 +284,9 @@ func createPVSpec(name string, readOnly bool, zones []string) *volume.Spec {
 	}
 
 	if zones != nil {
-		zonesLabel := strings.Join(zones, kubeletapis.LabelMultiZoneDelimiter)
+		zonesLabel := strings.Join(zones, volumeutil.LabelMultiZoneDelimiter)
```

Comment:
I realize gce already depends on volumeutil, so it hasn't made things significantly worse. However volumeutil is not part of staging. So we have clearly moved something (LabelMultiZoneDelimiter) which needed to be moved into staging somewhere other than staging. Are we planning to either move it again or to move volumeutil into staging?